sample-tracking-status-overview-domain/src/main/groovy/life/qbic/business/samples/count/CountSamples.groovy

```diff
@@ -30,65 +30,30 @@ class CountSamples implements CountSamplesInput{
     this.output = output
   }
 
-  /**
-   * This method calls the output interface with the number of all samples in a project as
-   * well as the subset of samples received by a lab.
-   * In case of failure the output interface failure method is called.
-   * @since 1.0.0
-   */
   @Override
-  void countReceivedSamples(String projectCode) {
+  void countSamplePerStatus(String projectCode) {
```

Comment:
```suggestion
  void countSamplesPerStatus(String projectCode) {
```
A small suggestion. You are counting the samples so the method should be called countSamplesPerStatus. Other than that, this is a very nice method name!

Reply:
I like your suggestion, I added it manually to make the refactoring easier. I hope this is ok for you:)